source/python/gui/source/__main__.py

```diff
@@ -162,6 +172,15 @@ def main():
         help="Do not launch the GUI, print the causal analysis out to the console",
     )
 
+    my_parser.add_argument(
```

Comment:
This should also be configurable in the GUI

Reply:
Do you mean for there to be the same CLI output to come out for GUI? I did that by using the verbosity flag.

Reply:
No, I mean a drop-down menu item in the GUI that lets you change the mode.

Reply:
Let's defer this to another PR